tasks/update-version.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * A task that updates version numbers in project files in accordance with
+ * build config.
+ *
+ * @param {Object} grunt The grunt helper object.
+ * @returns {void}
+ */
+module.exports = function( grunt ) {
+	grunt.registerMultiTask(
+		"update-version",
+		"Updates the version tags in all the right places.",
+		function() {
+			let options = this.options(
+				{
+					version: "version",
+					regEx: "regEx",
+					preVersionMatch: "preVersionMatch",
+					postVersionMatch: "postVersionMatch"
+				}
+			);
+			// foreach file in this.files
+			for ( let i = 0, n = this.files.length; i < n; i++ ){
```

Comment:
Please refrain from doing multiple variable assignments in loops. `i` is fine as it's part of the looping mechanism, but doing the additional `n` assignment is not. If you want to additionally improve performance, move the `this.files.length` out of the for loop and give it a better variable name; abbreviations should be used very sparsely.